js/data.js

```diff
@@ -0,0 +1,74 @@
+const AVATAR_INDEX = {
```

Comment:
в принципе, можно делать так:
```suggestion
export const AVATAR_INDEX = {
```
чтобы внизу не писать еще раз что надо экспортировать

Reply:
и так для каждой константы, но это не принципиально. Тут надо смотреть как принято на проекте, тк фронты могут договорить как писать, а могут и нет, тогда без разницы